pkg/kubectl/run.go

```diff
@@ -77,6 +80,22 @@ func (BasicReplicationController) Generate(genericParams map[string]interface{})
 		}
 		delete(genericParams, "args")
 	}
+
+	var envs []api.EnvVar
```

Comment:
This can be a todo, but we should abstract this logic so that multiple generators can handle env in the same way.  Same for parse envs.  This method is getting pretty long.